gwcs/coordinate_frames.py

```diff
@@ -279,6 +278,10 @@ def coordinate_to_quantity(self, *coords):
         Given a rich coordinate object return an astropy quantity object.
         """
         # NoOp leaves it to the model to handle
+        # If coords is a 1-tuple of quantity then return the element of the tuple
+        # This aligns the behavior with the other implementations
+        if not hasattr(coords, 'unit') and len(coords) == 1:
+            return coords[0]
```

Comment:
I'm trying to understand where this is a problem. Within gwcs there are two calls to `coordinate_to_quantity` and both pass  `*coords` . Are you calling it outside gwcs?

Also the new test does not actually test this use case.

Reply:
If you can change the test I htin kit's good to go

Reply:
I have added a test exercising this. I also reproduced the scenario in which I was hitting it just to be safe :rofl: